Slow Sort/README.markdown

```diff
@@ -17,19 +17,17 @@ We can decompose the problem of sorting n numbers in ascending order into
 Here is an implementation of slow sort in Swift:
 
 ```swift
-public func slowsort(_ i: Int, _ j: Int) {
-    if i>=j {
-        return
-    }
+func slowSort(_ i: Int, _ j: Int, _ numberList: inout [Int]) {
```

Comment:
Mind updating the indentation to 2 spaces? This function seems to be indented at 4 spaces.

Reply:
It's just in the README and it appears fine.
![screen shot 2018-10-26 at 6 51 38 pm](https://user-images.githubusercontent.com/44420971/47596984-4307dd00-d950-11e8-981b-40456146eb2a.png)

Reply:
@laurentaylormarshall We have a convention where all code should be indented for 2-spaces, that example uses 4 spaced indents.